docs/guides/iam_auth_rds_pg.md

```diff
@@ -246,7 +246,7 @@ module "cyral_sidecar" {
   source = "cyralinc/sidecar-ec2/aws"
 
   # Use the module version that is compatible with your sidecar.
-  version = "~> 5.0"
+  version = "~> 4.3"
```

Comment:
Some of these not related to current change, got generated and caused diff, can revert - just not sure if this is the correct info - or if the correct info is in main...

Reply:
Please rebase your PR from the HEAD of `main`. It should fix all these issues.

Reply:
NCM. Just saw the PR got `main` merged on top of it, so problem is now fixed.